### PR TITLE
[WIP] Add: new insights docs.

### PR DESCRIPTION
**Pull request by teesloane**
Add documentation on new insights features. 

- I also removed the sumo logic information - I feel like it was tacked on to this document a bit and that perhaps we can find a more fitting place for it?

### CODE REVIEW

**Review by thomasr8 — APPROVED** [review]
Great work @teesloane . Looks really good 👍

File: jekyll/_cci2/insights.md [hunk 1]
```diff
@@ -6,130 +6,88 @@ description: "Viewing the status of repos and test performance"
...
-- Status of all your repos building on CircleCI in real time
-
-- Median queue time
-
-- Median build time
 
-- Number of branches
+The CircleCI Insights feature provides a dashboard for viewing the health and
+usage of your repositories build processes. _Insights_ provides time-series data
+overviews of credit usage, success rates, pipeline duration, and other pertinent
```
Comment (millerpeterson) [inline jekyll/_cci2/insights.md:17]: We provide data only at the workflow and job levels.

File: jekyll/_cci2/insights.md [hunk 2]
```diff
@@ -6,130 +6,88 @@ description: "Viewing the status of repos and test performance"
...
+information.
 
-- Last build
+This document describes how to access the Insights feature on CircleCI Cloud and Server.
```
Comment (millerpeterson) [inline jekyll/_cci2/insights.md:20]: Insights 1.0 is now (mostly) only on Server, while Insights 2.0 is only on Cloud

**Review by millerpeterson — APPROVED** [review]
A few small notes. Also: there will be another page available at GA, but this could be addressed in a forthcoming PR.

**Pull request merged**